README.md

```diff
@@ -13,36 +13,44 @@
 ## Installation
 
 ```bash
-$ npm install -S @adobe/helix-auth-fastly
+$ npm install @adobe/helix-auth-fastly
 ```
 
 ## Usage
-helix-auth-fastly is a wrapper around actions to ensure that actions are fastly authenticated.
 
-helix-auth-fastly also exposes functionality for conditional fastly authentication.
+### As a wrapper
 
-To use helix-auth-fastly wrapper execute:
-```
-const { fastlyWrapper } = @adobe/helix-auth-fastly; 
+`helix-auth-fastly` is a wrapper around OpenWhisk actions to that ensures that invocations of the action are only successful if valid Fastly credentials, in the forms of a service ID and token, are provided.
+
+`helix-auth-fastly` can also be used as a simple single-purpose function to validate a pair of Fastly credentials.
+
+To use the `helix-auth-fastly` wrapper set up your OpenWhisk action like this:
+
+```javascript
+const { wrap } = require('@adobe/openwhisk-action-utils'};
+const { fastlyAuthWrapper } = require('@adobe/helix-auth-fastly'); 
 
-let action = func;
+const main = () = {
+  // your action code goes here
+};
 
-action = fastlyWrapper(action,
-{
-    token: fastlyAuthToken, 
-    service: fastlyServiceId,
-});
+module.exports.main = = wrap(action)
+  .with(fastlyAuthWrapper, {token: 'tokenName', service: 'serviceName'});
```

Comment:
I would use a more explanatory name, like `FASTLY_TOKEN` and `FASTLY_SERVICE`

Reply:
We typically reserve the all caps parameters for action-static parameters. The credentials provided to auth-fastly are typically not static, so I think the camel case example is fine.

Reply:
> The credentials provided to auth-fastly are typically not static

- why are they not static?
- convention is, that all _secrets_ are all caps...

Reply:
1. They are not static because actions like `helix-perf`, `helix-publish`, and `helix-logging` need to work with any service config and credential pair that has been provided by `helix-cli`, not just a statically configured one.
2. Again, `helix-perf`, `helix-publish`, and `helix-logging` which are the target users of the library use lower case parameters. We can change this there (breaking) and then update it here, but as it stands, the documentation represents the most likely use case.